src/main/controllers/CftLoginController.ts

```diff
@@ -0,0 +1,10 @@
+import {PipRequest} from "../models/request/PipRequest";
+import {Response} from "express";
+import {CFT_IDAM_URL, FRONTEND_URL} from '../helpers/envUrls';
+
+export default class CftLoginController {
+
+  public async get(req: PipRequest, res: Response): Promise<void> {
+    res.redirect(CFT_IDAM_URL + '?client_id=app-pip-frontend&response_type=code&redirect_uri=' + encodeURI(FRONTEND_URL + '/cft-login/return'));
```

Comment:
Might be a bit clearer to define the URL parameters separately like below:

    const params = new URLSearchParams({
      'client_id': 'app-pip-frontend',
      'response_type': 'code',
      'redirect_uri': FRONTEND_URL + '/cft-login/return'
    });

    res.redirect(CFT_IDAM_URL + ? + params.toString());

Reply:
Updated :)